setup.py

```diff
@@ -90,10 +90,12 @@
         'requests>=2.25.1,<3.0',
         'rfc3986-validator>=0.1.1',
         'tornado>=6.1.0',
+        'typing-extensions>=3.10,<4',  # Cap from KFP
         'traitlets>=4.3.2',
         'urllib3>=1.26.5',
         'watchdog>=2.1.3',
         'websocket-client',
+        'wrapt>=1.11,<1.14',  # Add from asteroid
```

Comment:
I don't see `asteroid` in our dependencies, do we know what of our dependencies is pulling in `asteroid` and, if so, could that be added to the comment?

Reply:
still trying to track this sucker down....

Reply:
Just saw the update, thanks.